libs/react/src/lib/in-page-wizard/inPageWizardStepCard.stories.mdx

```diff
@@ -1,5 +1,9 @@
-import { Meta, Story, Canvas } from '@storybook/addon-docs'
+import { ArgsTable, Meta, Story, Canvas } from '@storybook/addon-docs'

```

Comment:
Used ArgsTable plus documentation on props instead of handwriting the props table